widgetry/src/style/mod.rs

```diff
@@ -101,20 +101,20 @@ impl ButtonStyle {
     pub fn solid_primary() -> Self {
         Self {
             fg: hex("#F2F2F2"),
-            fg_disabled: hex("#F2F2F2").alpha(0.3),
-            bg: hex("#EE702E").alpha(0.8),
+            fg_disabled: hex("#F2F2F2"),
+            bg: hex("#EE702E").tint(0.1),
```

Comment:
Transparency makes things hard. In particular for small controls with text on them, like buttons.

I think we're going to be better served by mixing in white/black to achieve the different variations of a color, that isolates us from the complexity of the background affecting the text contrast.